riotfile.py

```diff
@@ -971,6 +971,47 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pkgs={"urllib3": ["~=1.22.0", ">=1.23,<1.27", latest]},
             command="pytest {cmdargs} tests/contrib/urllib3",
         ),
+        Venv(
+            name="aiohttp",
+            command="pytest {cmdargs} tests/contrib/aiohttp",
+            pkgs={"pytest": [">=3"], "pytest-aiohttp": [latest]},
+            venvs=[
+                Venv(
+                    pys=["3.6"],
```

Comment:
should this include py 3.5 as well?

Reply:
somehow all the tests were being skipped with 3.5, even with tox 🤷

Reply:
can we leave a comment here, otherwise someone might try to add 3.5 in the future